src/screens/SettingsScreen.tsx

```diff
@@ -56,47 +42,114 @@ export const SettingsScreen = () => {
             title={t('settings-profile-row-title', 'Profile')}
             action={() => navigate('Profile')}
           />
+          <Divider />
           <MainMenuItem
             title={account?.name || t('settings-account-selection', 'Accounts')}
             action={() => navigate('AccountSelection')}
           />
-          <View style={styles.subMenuContainer}>
-            <OAuthLogoutButton label={t('settings-logout', 'Logout')} />
-            <View style={styles.versionContainer}>
-              <Text testID={tID('version-text')}>
-                {t('settings-version', 'Version {{ version }}', {
-                  version: fullVersion,
-                })}
-              </Text>
-            </View>
-          </View>
+          <Divider />
         </ScrollView>
+        <View style={styles.subMenuContainer}>
+          <OAuthLogoutButton label={t('settings-logout', 'Logout')} />
+          <View style={styles.versionContainer}>
+            <Text testID={tID('version-text')}>
+              {t('settings-version', 'Version {{ version }}', {
+                version: fullVersion,
+              })}
+            </Text>
+          </View>
+        </View>
       </SafeAreaView>
     </View>
   );
 };
 
-const styles = StyleSheet.create({
-  mainMenuItem: {
+interface Props {
+  title: string;
+  action?: () => void;
+  badge?: React.ReactNode;
+}
+
+function MainMenuItem({ title, action, badge }: Props) {
+  const { styles } = useStyles('SettingsScreen', defaultStyles);
+
+  return (
+    <TouchableOpacity onPress={action} accessibilityRole="button">
+      <View style={styles.mainMenuItem}>
+        <Text style={styles.mainMenuItemText}>{title}</Text>
+        {badge}
+        <Text style={styles.arrow}>{'>'}</Text>
```

Comment:
TODO: Replace this with default actual icons. Possibly from react native paper.  Also maybe make an optional icon prop.

Reply:
It might be better to either put that TODO in code OR just leave this out for now.